guides/common/images/foreman_iss_disconnected_airgapped.drawio

```diff
@@ -1 +1 @@
-<mxfile host="Electron" modified="2022-05-17T23:20:08.954Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/15.3.8 Chrome/91.0.4472.164 Electron/13.5.0 Safari/537.36" etag="nynIQqBX6ozz9S7jC-y8" version="15.3.8" type="device"><diagram id="uq4QgXehMySjV9am80Ny" name="Page-1">7Vhdb9owFP01eWRK7PD1WEKh0sRWCY1Nfams2CUeThw5psB+/ZzGDvmifAxYtQE8xOfenMT33GMnWNAL12OB4mDCMWEWsPHagkMLAMcFwEp/Nt5kSM+GGTAXFOukLTClv4gGbY0uKSZJKVFyziSNy6DPo4j4soQhIfiqnPbCWfmqMZqTGjD1Eauj3ymWgUY7tr0NPBA6D/Sl+yYQIpOsgSRAmK8KELy3oCc4l9lRuPYIS4tn6pKdN9oRzW9MkEgecsLz55+z7uLH0+Bp6S3wpAt7wGlpllfElnrCHo9kypjds9yYQgi+jDBJuRwLDlYBlWQaIz+NrpT0CgtkyHT4RZFoLYFrxprLxD3OuHjjhjZoD3teilPGDB7xSKUPEin4gjQlZxGjiqrnoF4SMz8iJFkXIF2iMeEhkWKjUnQUGrl0v7p6uCqIb1KCgu4mD+l+m+fMW0nUgVblCIVATaERFyREkQKnRKiZ/a9SqbWkJJUyZU0rYF9VK7cmVk0cEuG7dF1SI5+hJKH+Tj0c9536n6XYCV8Kn+xfHyQScyL3dynBpdW0Ll1BmnaDMgYThCFJX8trcJNc+gqPnKZLVm5it9wZsKp4Nm99VnHNrBL1y0SgXyHKClMjeuuefNp/0FDtf62hPlajuM4efQ9tFLe6FvUu1ijQWcxm3/o9nHxt+9GXcLIYP7TgbZc4cJfIN/hr7BKNWjn9o0yty3ayo4v1xCgJcsnTwSOSkojoDQE2aJKn76VfFVH3NKLpVIf2e9J8FGtXDAlOdHYX7CG6sLNB4yN6+q6jhNztafsoTzudc3q6BJ7ZzhVZe1d8PG+Wp/58fjPzpc2saD/ZxQ88j7f38V7a6vVd/I6K1hjF8c3rhV3675n9uPe7m9nPYvZuxZSdM5l9D+/JZlfD7b96Wfr2v1F4/xs=</diagram></mxfile>
\ No newline at end of file
+<mxfile host="Electron" modified="2022-07-25T12:07:33.270Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/15.3.8 Chrome/91.0.4472.164 Electron/13.5.0 Safari/537.36" etag="XTg86G4ejI-tbtDDhGEh" version="15.3.8" type="device"><diagram id="uq4QgXehMySjV9am80Ny" name="Page-1">7Vhdb9owFP01eWRK7PD1WEKh0sRWCY1Nfams2CUeThw5psB+/ZzGDvmifAxYtQE8xOfenMT33GMnWNAL12OB4mDCMWEWsPHagkMLAMcFwEp/Nt5kSM+GGTAXFOukLTClv4gGbY0uKSZJKVFyziSNy6DPo4j4soQhIfiqnPbCWfmqMZqTGjD1Eauj3ymWgUY7tr0NPBA6D/Sl+yYQIpOsgSRAmK8KELy3oCc4l9lRuPYIS4tn6pKdN9oRzW9MkEgecsLz55+z7uLH0+Bp6S3wpAt7wGlpllfElnrCHo9kypjds9yYQgi+jDBJuRwLDlYBlWQaIz+NrpT0CgtkyHT4RZFoLYFrxprLxD3OuHjjhjZoD3teilPGDB7xSKUPEin4gjQlZxGjiqrnoF4SMz8iJFkXIF2iMeEhkWKjUnQUGrl0v7p6uCqIb1KCgu4mD+l+m+fMW0nUgVblCIVATaERFyREkQKnRKiZ/a9SqbWkJJUyZU0rYF9VK7cmVk0cEuG7dF1SI5+hJKH+Tj0c9536n6XYCV8Kn+xfHyQScyL3dynBpdW0Ll1BmnaDMgYThCFJX8trcJNc+gqPnKZLVm5it9wZsKp4Nm99VnHNrBL1y0SgXyHKClMjeuuefNp/0FDtf62hPlajuM4efQ9tFLe6FvUu1ijQWcxm3/o9nHxt+9GXcLIYP7TgbZc4cJfIN/hr7BKNWjn9o0yty3ayo4v1xCgJcsnTwSOSkojoDQE2aJKn76VfFVH3NKLpVIf2e9J8FGtXDAlOdHYX7CG6sLNB4yN6+q6jhNztafsoTzudc3q6BJ7ZzhVZe1d8PG+Wp/58fjPzpc2saD/ZxQ88j7f38V7a6vVd/I6K1hjF8c3rhV3675n9uPe7m9nPYvZuxZSdM5l9D+/JZlfD7b96Wfr2v1F4/xs=</diagram></mxfile>
```

Comment:
So all you did was add the (white) background instead of "transparent"? If so, this change is OK for me!

Reply:
I think I originally changed the keyword "Disconnected" to "Air-Gapped" plus the background, but it got lost somewhere during cherry-picking perhaps. :D So, yes :D